src/dxt.rs

```diff
@@ -33,7 +33,7 @@ impl DXTVariant {
     /// that is encoded in a single DXTn block
     fn decoded_bytes_per_block(self) -> usize {
         match self {
-            DXTVariant::DXT1 => 48,
+            DXTVariant::DXT1 => 64,
             DXTVariant::DXT3 | DXTVariant::DXT5 => 64,
```

Comment:
Hmm, reading [the spec](https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_texture_compression_s3tc.txt) a bit more, there is apparently out of band information as to whether DXT1 data is RGB or RGBA. Either way, for the color0 <= color1 case, the last item in the LUT is defined as R=G=B(=A)=0.

Reply:
The big question here is whether distinguishing between DXT1 with and without alpha is wanted on an API level, and how. A clever encoder could notice that an image has (almost) pure black but no transparency, and use the implicit black in the palette to be able to represent a wider range of colors. (i.e. 3 colors interpolated between the two bright endpoints and a disconnected black vs 4 colors interpolated between the endpoints, one of which would have to be black)

Of course it would also be possible to always decode to RGBA and discard the alpha channel afterwards (in user code?) if necessary, but that sounds rather inelegant here.

Reply:
Perhaps add a `DXTVariant::DXT1_A` or something?